libs/sdk-ui-kit/src/Header/HeaderWorkspacePicker.tsx

```diff
@@ -38,6 +38,7 @@ export interface IHeaderWorkspacePickerProps {
     onSelect?: (item: IHeaderWorkspace) => void;
     onScrollEnd?: (visibleRowsStartIndex: number, visibleRowsEndIndex: number) => void;
     projectPickerFooter?: React.ReactNode;
+    isRenamingProjectToWorkspaceEnabled?: boolean;
```

Comment:
It seems that file https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/api/sdk-ui-kit.api.md has not been updated yet to include this change.
→Please build the sdk to update `sdk-ui-kit.api.md`.